Remotion/Data/DomainObjects.UnitTests/UnitTestStorageProviderStub.cs

```diff
@@ -54,43 +54,40 @@ public void Dispose ()
 
     private static int s_nextID = 0;
 
-    private static readonly SafeContextSingleton<StorageProvider> _innerMockStorageProvider =
-        new SafeContextSingleton<StorageProvider>(typeof(UnitTestStorageProviderStub) + "._innerMockStorageProvider", () => null);
+    private static readonly SafeContextSingleton<IStorageProvider> _innerMockStorageProvider =
+        new SafeContextSingleton<IStorageProvider>(typeof(UnitTestStorageProviderStub) + "._innerMockStorageProvider", () => null);
 
-    public static IDisposable EnterMockStorageProviderScope (StorageProvider mock)
+    public static IDisposable EnterMockStorageProviderScope (IStorageProvider mock)
     {
       var previous = _innerMockStorageProvider.Current;
       _innerMockStorageProvider.SetCurrent(mock);
       return new MockStorageProviderScope(previous);
     }
 
-    public static T ExecuteWithMock<T> (StorageProvider mockedStorageProvider, Func<T> func)
+    public static T ExecuteWithMock<T> (IStorageProvider mockedStorageProvider, Func<T> func)
     {
       using (EnterMockStorageProviderScope(mockedStorageProvider))
       {
         return func();
       }
     }
 
-    public static StorageProvider CreateStorageProviderMockForOfficial ()
+    public static IStorageProvider CreateStorageProviderMockForOfficial ()
     {
       var storageProviderDefinition = MappingConfiguration.Current.GetTypeDefinition(typeof(Official)).StorageEntityDefinition.StorageProviderDefinition;
-      return new Mock<StorageProvider>(storageProviderDefinition, NullPersistenceExtension.Instance).Object;
+      return new Mock<IStorageProvider>(storageProviderDefinition, NullPersistenceExtension.Instance).Object;
     }
```

Comment:
It looks like dead code to me? (no usages for CreateStorageProviderMockForOfficial)